bin/build

```diff
@@ -12,3 +12,10 @@ source "$bp_dir/lib/build.sh"
 if check_tsc_binary "$build_dir" ; then
   export PATH=./node_modules/typescript/bin:$PATH
 fi
+
+# shellcheck disable=SC2034
+if detect_tsconfig_env_var ; then
+  tsconfig="$CUSTOM_TSCONFIG"
+else
+  tsconfig="tsconfig.json"
+fi
```

Comment:
I think we can just make this something like:
```sh
TSCONFIG=${CUSTOM_TSCONFIG:-"tsconfig.json"}
```
(https://stackoverflow.com/a/12691027/4135520)

Reply:
wow so nifty, that simplified everything lol